specification/communication/data-plane/MediaComposition/preview/2022-07-16-preview1/mediacomposition.json

```diff
@@ -0,0 +1,1180 @@
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Update Media Composition - Change source": {
+            "$ref": "./examples/MediaComposition_UpdateMediaComposition.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "MediaComposition"
+        ],
+        "summary": "Deletes a media composition",
+        "operationId": "MediaComposition_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mediaCompositionId",
+            "description": "",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "400": {
+            "description": "Bad Request.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Delete Media Composition": {
+            "$ref": "./examples/MediaComposition_DeleteMediaComposition.json"
+          }
+        }
+      }
+    },
+    "/mediaCompositions/{mediaCompositionId}/:start": {
```

Comment:
The recommended pattern for performing an action is
`.../{mediaCompositionId}:start` without a leading `/` in front of the colon. I noticed that there are other ACS services that follow the `/:` pattern. However, these also appear to be in preview. It would be good to change these if possible.